pandamarket.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang='ko'>
+    <head>
+        <meta charset="utf-8">
+        <title>판다마켓</title>
+        <link rel="stylesheet" href="pandamarket.css">
+    </head>
+    <body>
+       <header>
+        <div class='navbar-container'>
+          <a href="pandamarket.html">
+           <img class='logo' src='./img/logo.png' alt='판다마켓로고'>
```

Comment:
💊 제안  
img 태그의 alt 속성은 alternative 라는 의미로, 이미지 파일을 다운로드하는 것에 실패해서 이미지 파일을 보여줄 수 없을 때  
어떤 이미지인지 파악할 수 있게 대신 제공되거나, 스크린리더로 읽혀지는 문자를 의미합니다.  
따라서 가능한 해당 이미지를 잘 표현할 수 있는 문자열을 넣어주는 것이 좋습니다.


```suggestion
           <img class='logo' src='./img/logo.png' alt='판다마켓'>
```